gevent/select.py

```diff
@@ -3,11 +3,18 @@
 from gevent.event import Event
 from gevent.hub import get_hub
 from gevent.hub import integer_types
+from select import POLLIN, POLLOUT
 
-__implements__ = ['select']
+try:
+    from select import poll as original_poll
+except ImportError:
+    original_poll = None
+
+__implements__ = ['select', 'poll']
```

Comment:
If the original `select` module didn't implement `poll`, we probably shouldn't either.